### DISCUSSION TITLE
Auto Pip Like "ARC" & "OPERA"

### DISCUSSION THREAD

**Discussion by knownIndie** [zen-browser/desktop#996] [Ideas]
### What happened?

Well i like the browser and have been using it for a month and the only thing i'm missing right now is the support for __autopip__  

if this feature gets enabled with some playback tools its going to make this browser better

### Reproducible?

- [x] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.0

### Severity impact

Low

### What platform are you seeing the problem on?

macOS - aarch64

**Comment by nocanoa** [comment]
Will come soon from upstream

**Comment by Donnnno** [comment]
https://blog.nightly.mozilla.org/2024/08/26/streamline-your-screen-time-with-auto-open-picture-in-picture-and-more-these-weeks-in-firefox-issue-166/

**Comment by knownIndie** [comment]
if an example is required you can check out Arc and opera browsers

**Comment by Donnnno** [comment]
Not necessary. Firefox will implement this in its next major release, and after that Zen will have it too.

**Reply by tobias-hay** [reply]
Do you know if said major release has happened yet? I am assuming not considering auto pip does not seem to be a thing that's happening.

**Reply by vraravam** [reply]
Its still an experimental/lab feature in core FF. I assume that once its released in FF stable, then it can make its way into Zen.

**Comment by JSGRD22** [comment]
It's already present as an optional feature in firefox labs. You can turn it on right now. The only problem is the captions don't work properly.

**Reply by tobias-hay** [reply]
Yea, I enabled that but doesn't seem to work for me :/

**Comment by PantiesIsStoopid** [comment]
You can go into about:config and set `media.videocontrols.picture-in-picture.enable-when-switching-tabs.enabled` to `true`.I agree that this flag should be a checkbox in settings though as it is not obvious that such a flag exists.
